WordPress/src/main/java/org/wordpress/android/ui/sitecreation/domains/compose/DomainItem.kt

```diff
@@ -0,0 +1,59 @@
+package org.wordpress.android.ui.sitecreation.domains.compose
+
+import android.content.res.Configuration
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material.Divider
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import org.wordpress.android.ui.compose.theme.AppTheme
+import org.wordpress.android.ui.sitecreation.domains.SiteCreationDomainsViewModel.ListItemUiState.New.DomainUiState
+
+private val CostColorDefault @Composable get() = MaterialTheme.colors.onSurface.copy(alpha = 0.6f)
+
+@Composable
+fun DomainItem(
+    uiState: DomainUiState,
+) = with(uiState) {
+    Column {
+        Row(
+            modifier = Modifier
+                .clickable { onClick.invoke() }
+                .padding(horizontal = 24.dp, vertical = 16.dp)
```

Comment:
> ❓ Question: I know we have a [Margin](https://github.com/wordpress-mobile/WordPress-Android/blob/7eca2b306f664b28c591de523974acbabfe0c920/WordPress/src/main/java/org/wordpress/android/ui/compose/unit/Margin.kt) class for some commonly used margin values, I wonder if it's worth using that or if it doesn't make sense for Compose. Wdyt?

I agree with you, although there are other incoming changes which will most probably remove or change these margins (see task: #17985), and because of that my reasoning was that it's most simple to just use the actual values for now.

I would normally use the `Margin` class although my long term goal for it would be to turn it into an inline class but that's another topic not for now 👍 

> Since we usually get the actual dimension value in Figma design specs instead of "tokens" (like: "use a medium margin" or "extra-large margin") I am not 100% sure what is the best approach.

On a sidenote I keep thinking about this as well and dimensions aren't even the greatest issue I see when it comes to consistency in designs 😅 . Colors and typography are even more variable, for them almost every design in Figma redefines like 3+.

I would actually be in favor of just going with actual values and gradually stop creating so many "standards" in code for UI which ultimately change in the blink of an eye with every new design and end up being used only 2 times or more. Perhaps being flexible is the better approach here, and if that's to be our goal, then IMHO there's more flexibility when just using the direct values.

As a general rule, the way I reason about this is that something has to be used at least 2 times to make it a variable or function, otherwise unless it helps make the code more readable, inlining is better than extracting IMHO.